src/app/secure-data-environment/resources/organisation.resources.ts

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2022-2023 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+import { Uuid } from '@maurodatamapper/mdm-resources';
+
+export interface Organisation {
+  id: Uuid;
+  createdAt: Date;
```

Comment:
No action for now, but we will probably need to think about Date representation in JSON DTOs because it definitely won't be a JavaScript Date

Reply:
Having looked at this, we should use string, namely the format used by JavaScripts Date.toJSON() = '2012-04-23T18:25:43.511Z'